test/pg-cdc/pg-cdc.td

```diff
@@ -192,6 +192,27 @@ contains:publication "no_such_publication" does not exist
     "space table"
   );
 
+> SHOW SOURCES
+conflict_table        subsource <null>
+escaped_text_table    subsource <null>
+large_text            subsource <null>
+multipart_pk          subsource <null>
+mz_source             postgres  1
+no_replica_identity   subsource <null>
+nonpk_table           subsource <null>
+nulls_table           subsource <null>
+pk_table              subsource <null>
+"space table"         subsource <null>
+trailing_space_nopk   subsource <null>
+trailing_space_pk     subsource <null>
+types_table           subsource <null>
+utf8_table            subsource <null>
+таблица               subsource <null>
+
+# Dependency graph is that mz_source depends on all subsources
+! DROP SOURCE conflict_table
```

Comment:
Can you add a test that does `drop source conflict_table cascade;` and confirms that the source and all other sub-sources are dropped. A separate .td file may be required.